src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1204,14 +1204,16 @@ internal SingleLookupResult CheckViability(Symbol symbol, int arity, LookupOptio
 
             bool IsBadIvtSpecification()
             {
+                IEnumerable<ImmutableArray<byte>> keys = unwrappedSymbol.ContainingAssembly.GetInternalsVisibleToPublicKeys(this.Compilation.AssemblyName);
```

Comment:
You have to do this inside the `if` or, as the comment says, we'll stack overflow when binding attributes.

Reply:
pushed